pywatts/summaries/f1_summary.py

```diff
@@ -0,0 +1,67 @@
+import logging
+from typing import Dict, Optional, List, Tuple
+
+import numpy as np
+import pandas as pd
+
+from pywatts.summaries.metric_base import MetricBase
+from sklearn.metrics import f1_score
+
+logger = logging.getLogger(__name__)
+
+
+class F1Score(MetricBase):
+    """
+    Module to calculate the F1Score
+
+    :param offset: Offset, which determines the number of ignored values in the beginning for calculating the MAE.
+                   Default 0
+    :type offset: int
+    :param filter_method: Filter which should performed on the data before calculating the MAE.
+    :type filter_method: Callable[[np.ndarray, np.ndarray], Tuple[np.ndarray, np.ndarray]]
+    :param cuts: A list of Tuple of pd.Timestamps which specify intervals on which the metric should be calculated.
+    :type cuts: List[Tuple[pd.Timestamp, pd.Timestamp]]
+    :param average: The average param for the f1 score sklearn implementation
+    :type average: str
+    """
+
+    def __init__(self, name: str = "F1Score", average="micro",  **kwargs):
```

Comment:
Are the params filter_method and cuts included in **kwargs?

Reply:
Yes. Both are passed to the super class since they are needed there.